src/Command/Status.php

```diff
@@ -71,7 +74,9 @@ public function execute(InputInterface $input, OutputInterface $output)
     {
         try {
             $this->showStatus->run($input, $output);
-            $output->writeln('<info>' . $this->magentoVersion->get() . '</info>');
+            if ($input->getOption('format') !== 'json') {
+                $output->writeln('<info>' . $this->magentoVersion->get() . '</info>');
+            }
```

Comment:
```suggestion
            if ($input->getOption('format') !== 'json') {
                $output->writeln('<info>' . $this->magentoVersion->get() . '</info>');
            }
```
@peterjaap  Better to move this code to `\Magento\CloudPatches\Command\Process\ShowStatus` class along with `\Magento\CloudPatches\Composer\MagentoVersion` dependency. In this case you also need to update constructors   in `\Magento\CloudPatches\Test\Unit\Command\Process\ShowStatusTest` and `\Magento\CloudPatches\Test\Unit\Command\StatusTest`

Reply:
Done!

Reply:
Thanks, please update unit tests \Magento\CloudPatches\Test\Unit\Command\Process\ShowStatusTest and \Magento\CloudPatches\Test\Unit\Command\StatusTest because currently they are failing after moving \Magento\CloudPatches\Composer\MagentoVersion dependency

Reply:
No thanks. You can do it if you'd like, you were the one that suggested the change.

I'll just close this PR otherwise, it has taken me too much time already for such a little feature.

And people wonder why it takes ages to merge something into the Magento codebase...

Reply:
@peterjaap no problem, I can help you with the unit tests update. It's not about Magento, you will not be able to merge a PR to any other OS project with red builds. The requirement for test coverage is in DoD (definition of done), sorry.